torchao/_models/llama/benchmarks.sh

```diff
@@ -1,44 +1,28 @@
 export CHECKPOINT_PATH=../../../checkpoints # path to checkpoints folder
```

Comment:
I think one improvement for this script might be to use a loop to only specify `quantization` or whatever arg that changes, instead of spelling out all the args, it might be easier to see what benchmarks we are running

Reply:
that would be good if we were productionizing things but most of the time i'm commenting things out and only running 1 or 2 examples and having them all spelt out is really nice